src/Configuration/src/RandomValue/Steeltoe.Configuration.RandomValue.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net8.0</TargetFrameworks>
-    <Description>Configuration provider for generating random values</Description>
-    <PackageTags>configuration;random values</PackageTags>
+    <Description>Configuration provider for generating random values.</Description>
+    <PackageTags>configuration;ConfigurationProvider;random;values</PackageTags>
```

Comment:
It looks like a replacement of spaces to semicolons was done without much attention to the changes. For example, "values" as a tag on its own isn't very useful.

Reply:
Honestly, I don't think it makes a whole lot of difference what we do here, particularly for words that are repeated